src/consensus/params.cpp

```diff
@@ -144,11 +184,11 @@ namespace Consensus {
             return FundingStreamError::ILLEGAL_RANGE;
         }
 
-        if (params.FundingPeriodIndex(startHeight, endHeight - 1) >= addresses.size()) {
+        if (params.FundingPeriodIndex(startHeight, endHeight - 1) >= recipients.size()) {
             return FundingStreamError::INSUFFICIENT_ADDRESSES;
```

Comment:
```suggestion
            return FundingStreamError::INSUFFICIENT_RECIPIENTS;
```

Reply:
Variant removed.

Reply:
This variant has not been removed in the current state of the PR, and it can occur for funding streams that don't have addresses, so the suggestion still applies.